src/datagen/main/java/mekanism/common/recipe/impl/MekanismRecipeProvider.java

```diff
@@ -1353,6 +1353,18 @@ private void addGearModuleRecipes(Consumer<FinishedRecipe> consumer) {
               .key(Pattern.HDPE_CHAR, MekanismTags.Items.PELLETS_POLONIUM)
               .key(Pattern.WOOD, Items.DIAMOND_PICKAXE)
               .build(consumer);
+        //Fortune Unit
+        ExtendedShapedRecipeBuilder.shapedRecipe(MekanismItems.MODULE_FORTUNE)
+              .pattern(RecipePattern.createPattern(
+                    TripleLine.of(Pattern.ALLOY, Pattern.CONSTANT, Pattern.ALLOY),
+                    TripleLine.of(Pattern.WOOD, Pattern.PREVIOUS, Pattern.WOOD),
+                    TripleLine.of(Pattern.HDPE_CHAR, Pattern.HDPE_CHAR, Pattern.HDPE_CHAR))
+              ).key(Pattern.ALLOY, MekanismTags.Items.ALLOYS_ELITE)
+              .key(Pattern.PREVIOUS, MekanismItems.MODULE_BASE)
+              .key(Pattern.CONSTANT, MekanismBlocks.REFINED_GLOWSTONE_BLOCK)
+              .key(Pattern.HDPE_CHAR, MekanismTags.Items.PELLETS_POLONIUM)
+              .key(Pattern.WOOD, Items.DIAMOND)
```

Comment:
Maybe netherite picks? Unsure given I don't really know what represents "fortune" well

Reply:
Hmm, change the pattern to `Pattern.DIAMOND` an then change `Items.DIAMOND` to `Tags.Items.STORAGE_BLOCKS_DIAMOND`. (To rerun datagen you may need to temporarily set the forge version in gradle.properties to `.31`)

Reply:
And actually if you change the `MekanismBlocks.REFINED_GLOWSTONE` to `MekanismTags.Items.STORAGE_BLOCKS_REFINED_GLOWSTONE` for both the fortune and silk touch unit that would be great, otherwise I will do it at some other point given it doesn't matter quite as much.